partner_multi_relation/models/res_partner_relation_type_selection.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# © 2014-2016 Therp BV <http://therp.nl>
+# © 2017 Therp BV <http://therp.nl>
```

Comment:
All the copyrights for existing files should be 2014-2017. Only the really new files should be 2017. Please check this everywhere.